wrapanapi/systems/rhevm.py

```diff
@@ -860,6 +860,27 @@ def list_cluster(self, **kwargs):
         cluster_list = self.api.system_service().clusters_service().list(**kwargs)
         return [cluster.name for cluster in cluster_list]
 
+    def list_disks(self, **kwargs):
+        disks_list = self.api.system_service().disks_service().list()
+        if kwargs:
+            disk_status = kwargs.get('status')
+            if disk_status:
+                disk_status = disk_status.upper()
+
+                if disk_status == 'OK':
+                    return [(disk.name, disk.status) for disk in disks_list
+                    if types.DiskStatus.OK == disk.status]
+
+                if disk_status == 'LOCKED':
+                    return [(disk.name, disk.status) for disk in disks_list
+                    if types.DiskStatus.LOCKED == disk.status]
+
+                if disk_status == 'ILLEGAL':
+                    return [(disk.name, disk.status) for disk in disks_list
+                    if types.DiskStatus.ILLEGAL == disk.status]
+
+        return [(disk.name, disk.status) for disk in disks_list]
```

Comment:
I think it would be good to return just the disk name, and provide a `get_disk(self, name)` method to then fetch further information.

Reply:
ok let me add that